### PR TITLE
 OCPBUGS-27507: bump containerd for vulnerability fix

### PR DESCRIPTION
**Pull request by r4f4**

### CODE REVIEW

**Review by barbacbd — COMMENTED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -308,7 +308,8 @@ replace github.com/prometheus/client_golang => github.com/prometheus/client_gola
 replace golang.org/x/net => golang.org/x/net v0.5.0
 
 // https://issues.redhat.com/browse/OCPBUGS-8119
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.5.18
+// https://issues.redhat.com/browse/OCPBUGS-27507
```
Comment (sadasu) [inline go.mod:311]: Nit: There are 2 bug fixes in the comments. Is that intended?

Comment (r4f4) [reply go.mod:311]: Yes, but I can drop the oldest if that's preferable.

**Pull request merged**